spec/rails/helpers_system_spec.rb

```diff
@@ -8,6 +8,19 @@
       %r{Select datetime:\s*<select.*name="date.*>.*<option.*value="2014".*</option>}mi)
   end
 
+  it 'should support built-in Rails helpers even when automatic_helper_access is false' do
+    expect_match("rails_helpers_without_automatic_helper_access",
+                 /Three months ago: 3 months/mi,
+                 /A million dollars: \$1,000,000\.00/mi,
+                 /class="debug_dump"/mi)
+  end
+
+  it 'should support url helpers even when automatic_helper_access is false' do
```

Comment:
@ageweke this spec isn't passing because I'm not exactly sure how to create routes within oop_rails_server. Can you point me in the right direction here?